cabal-install/Distribution/Deprecated/ParseUtils.hs

```diff
@@ -177,7 +177,7 @@ warning s = ParseOk [PWarning s] ()
 data FieldDescr a
   = FieldDescr
       { fieldName     :: String
-      , fieldGet      :: a -> Doc
+      , fieldGet      :: a -> [Doc]
```

Comment:
the `Distrubution.Fields.Pretty` cheats. It renders `Doc` and sees if it's a multiline, and then indents

https://github.com/haskell/cabal/blob/0d2d9ca7b7a1689dba9c3a22a8d9966b05b2957f/Cabal/Distribution/Fields/Pretty.hs#L107-L111

I'd use here the same approach, i.e. keep `fieldGet` as `a -> Doc`. It won't make migration *harder*.